### PR TITLE
Add google analytics using Nuxt.js GA package

### LINKED ISSUE #19: Add Google Analytics

### PR DESCRIPTION
**Pull request by MarkBennett**
Followed the directions on this page to add GA.

https://google-analytics.nuxtjs.org/setup

We might still need to add some custom tracking to the components as well depending what we decide to track.

Closes #19.

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ export default {
     buildModules: [
         // Doc: https://github.com/nuxt-community/nuxt-tailwindcss
         '@nuxtjs/tailwindcss',
+        '@nuxtjs/google-analytics',
```
Comment (stormwarning) [inline nuxt.config.js:39]: Maybe throw a comment above this with a link to the docs, just for handy reference?

**Comment by MarkBennett** [comment]
I added a link to the Nuxt GA docs @stormwarning. Mind taking another look?

File: nuxt.config.js (hunk 1 above)
Comment (MarkBennett) [reply nuxt.config.js:40]: Done!

**Review by stormwarning — APPROVED** [review]
👍

**Pull request merged**